Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache-2.0 OR MIT"
 readme = "README.md"
 repository = "https://github.com/paritytech/soketto"
 edition = "2021"
-rust-version = "1.56.1"
+rust-version = "1.71.1"
```

Comment:
It looks like env-logger only requires this but bumped it to be on the safe side.